cms/templates/js/no-content-groups.underscore

```diff
@@ -0,0 +1,3 @@
+<div class="no-group-configurations-content">
```

Comment:
Good point. However, there is no template in the non-empty case; children views are simply rendered into the parent's element. So I'm not sure if that refactor would work given the current implementation.